AstroCal/view/home.py

```diff
@@ -0,0 +1,157 @@
+from turtle import color
+import kivy
+import random
+
+from kivy import *
+from kivy.app import *
+from kivy.uix.label import *
+from kivy.core.window import *
```

Comment:
For some reason "from kivy.core.window import *" causes a window to automatically get created despite not calling any functions from home.py. This is an issue when we only want console output we get this blank window. 
![image](https://user-images.githubusercontent.com/58247240/194810572-68fa81bd-4cb0-45c0-8f8e-3b653f5c3787.png)

A fix should be made to prevent a window from appearing if we only want console output.